tests/cypress/e2e/teacher_page/customize_class_page/ind_adv_button.cy.js

```diff
@@ -0,0 +1,106 @@
+import { loginForTeacher } from '../../tools/login/login.js'
+import { createClass } from "../../tools/classes/class";
+
+describe('Testing individual adventure buttons', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+
+    cy.get('#class_view_button > .no-underline').click() // Press on view class button
+
+    cy.get('#customize-class-button').click(); // Press customize class button
+
+    // check every individual level button:
+    cy.get('#level_button_1').click()
+    cy.get('.adventure_level_1').should('be.not.checked')
+    cy.get('#level_button_1').click()
+    cy.get('.adventure_level_1').should('be.checked')
```

Comment:
Is there a way to make this a bit more dynamic? This is a lot of duplicate code for all levels.

Reply:
I was able to make this dynamic! I used a for loop, I think I can maybe also do this with the date inputs on this page.